scripts/semgrep.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eo pipefail
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
```

Comment:
They do not want us to run a build so the scan doesn't pick up dev deps

Reply:
ack